tools-ansible/inventory.ini

```diff
@@ -0,0 +1,33 @@
+
+# [syncer_servers]
+# # must specify job_name, and syncer_status_port. syncer_status_port must be same with status_addr(the port part) in the config file
+# syncer_1 ansible_host=172.16.30.130 job_name=1 syncer_status_port=10081
+
+# [loader_servers]
+# # must specify job_name, and loader_status_port. loader_status_port must be same with status_addr(the port part) in the config file
+# loader_1 ansible_host=172.16.30.130 job_name=1 loader_status_port=10084
+
+# [mydumper_servers]
+# # must specify job_name.
+# mydumper_1 ansible_host=172.16.30.130 job_name=1 
+
+[integrated_servers]
+integrated_1 ansible_host=172.16.30.130 job_name=1 syncer_status_port=10081 loader_status_port=10084 master_user="root" master_password="Pingcap1@#" master_host="127.0.0.1" master_port=3306 dump_db=foo
+
+## Global variables
+
+[all:vars]
+# we will use deploy_dir/data to save mydumper data, please ensure deploy_dir is large enough to hold the dump data.
+deploy_dir = /home/tidb/deploy
+# ssh via normal user
+ansible_user = tidb
+
+# timezone of deployment region
```

Comment:
remove useless variable:
+# timezone of deployment region
 +timezone = Asia/Shanghai
 +set_timezone = True
 +
 +enable_ntpd = True
 +set_hostname = False